python/paddle/fluid/dygraph/dygraph_to_static/program_translator.py

```diff
@@ -282,7 +293,13 @@ def from_func_spec(func_spec):
                 # 3. Builds program only once and returns the output Variables.
                 with param_guard(func_spec.parameters(False)), param_guard(
                         func_spec.buffers(False)):
-                    outputs = static_func(*inputs)
+                    try:
+                        outputs = static_func(*inputs)
+                    except BaseException as e:
+                        # NOTE: If e raised in compile time, e should be attached ERROR_DATA here.
```

Comment:
NIT (Not important): English grammar, "if e is raised in compile time, e should be attached to ERROR_DATA here.

Reply:
Thanks, done